### PR TITLE
Add Build only CI

### PR DESCRIPTION
**Pull request by lumjjb**
Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

**Review by mamy-CS — APPROVED** [review]
LGTM!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Tornjak CI
+on: [push]
+jobs:
+  tornjak-build:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
```
Comment (mrsabath) [inline .github/workflows/ci.yml:7]: I like the icons! :)

File: Makefile [hunk 2]
```diff
@@ -37,12 +37,18 @@ vendor:
 	go mod vendor
 
 container-agent: bin/tornjak-agent ui-agent
-	docker build --no-cache -f Dockerfile.add-frontend -t ${CONTAINER_TAG} . && docker push ${CONTAINER_TAG}
+	docker build --no-cache -f Dockerfile.add-frontend -t ${CONTAINER_TAG} .
+
+container-agent-push: container-agent
```
Comment (mrsabath) [inline Makefile:42]: That's nice that you decoupled the build from the push

**Review by mrsabath — APPROVED** [review]
LGTM

**Pull request merged**